### ISSUE TITLE
Trouble with Mac installation missing libusb

### ISSUE THREAD

**Issue by wpd** [bytedeco/javacv#501]
I downloaded and unzipped javacv-1.2-bin.zip as per README.md on my MacBook running OS X Mavericks.  When I try to run the example from the "Sample Usage" section in Eclipse, I get the following error:

```
dyld: lazy symbol binding failed: Symbol not found: _libusb_init
  Referenced from: /private/var/folders/4l/yrsmlwtx0mdfhww3vsql_fqc0000gn/T/javacpp43560497880605/libdc1394.22.dylib
  Expected in: flat namespace

dyld: Symbol not found: _libusb_init
  Referenced from: /private/var/folders/4l/yrsmlwtx0mdfhww3vsql_fqc0000gn/T/javacpp43560497880605/libdc1394.22.dylib
  Expected in: flat namespace
```

I tried installing libusb from MacPorts, and setting DYLD_LIBRARY_PATH and/or DYLD_FALLBACK_LIBRARY_PATH to point to /opt/local/lib (via Run Configuration environment variables) but nothing seems to help.

I feel like I am missing something very basic here.  Can anybody help me?

**Comment by saudet** [comment]
Looks like it doesn't get bundled... Try to install it with Homebrew.

**Comment by wpd** [comment]
Hi Saudet,
Thank you for the suggestion...I did install it with MacPorts.  How would Homebrew be any different?  Macports installs it in /opt/local/lib.

--wpd

**Comment by saudet** [comment]
BTW, if you do not need libdc1394, we can ignore this error.

**Comment by Skinner927** [comment]
I too am having this issue. I've tried libusb installs with both homebrew and ports. I'm not sure where to go from here.

**Comment by saudet** [comment]
@Skinner927 Do you need to use libdc1394??

**Comment by saudet** [comment]
In any case, this is fixed in version 1.3. Enjoy!
